content/04.results.md

```diff
@@ -73,11 +73,11 @@ That is, in many cases, matching the gene being studied with the "correct" data
 
 We additionally compared pan-cancer survival prediction performance using principal components derived from each data type; in general, results were comparable across the three data types (Figure {@fig:methylation}I).
 All data types outperformed the covariate-only baseline (see [Methods](#survival-prediction-using--omics-datasets)) for lower numbers of PC features included, although performance was similar to the baseline for higher numbers of PCs.
-Confidence intervals between the best- and worst-performing data types overlap at most PC counts (with the exception of gene expression at 5000 PC feature), suggesting that similarly to mutation prediction, the three data types tend to have comparable effectiveness for pan-cancer survival prediction.
+Confidence intervals between the best- and worst-performing data types overlap at most PC counts (with the exception of gene expression at 5,000 PC feature), suggesting that similarly to mutation prediction, the three data types tend to have comparable effectiveness for pan-cancer survival prediction.<!--Is there a word missing in the parenthetical? It reads a bit confusingly-->
 
 ![
 **A.** Count of overlapping samples between gene expression, 27K methylation, 450K methylation, and somatic mutation data used from TCGA.
-Only non-zero overlap counts are shown.
+Only non-zero overlap counts are shown.<!--Does this mean that the ones that look like 0 are > 0? Also, I think you used only a subset of these for your experiments, could you distinguish them somehow?-->
```

Comment:
```suggestion
Only non-zero overlap counts are shown.
```
Yes, the ones that look like 0 are small but >0. We didn't technically "use" the somatic mutation data (i.e. use it as predictive features like we did with the other data types), but because we're using mutation information to generate the labels/dependent variables we also have to consider whether our sample set overlaps with the mutation data in all of our experiments. I'll explain this briefly in the text or legends, in my next PR.